src/components/landscape-sidebar/geo-description-widget/geo-description-widget-component.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { Loading } from 'he-components';
-import ShareModal from 'components/share-modal';
 import { VIEW_MODE } from  'constants/google-analytics-constants';
 import styles from './geo-description-widget-styles.module.scss';
+import * as actions from 'actions/url-actions';
+import { connect } from 'react-redux';
+import FixedHeader from 'components/fixed-header'
+
+const GeoDescriptionWidget = (props) => {
+  const { data, loading, error, view, changeGlobe } = props;
+
+  const handleBackClick = () => {
+    const params = { zoom: 7 }
```

Comment:
If we want to make this method perform faster we can pass also the `center` of the actual view as a param. We can get the actual `center` from the `globe` `key` of the `URL`. Not really sure why it performs better but it does (the `center` is an optional parameter but seems that explicitly passing it to the method eases things out, maybe that way it has not to compute the actual `center` or `target` for the method :)